### PR TITLE
[Fixed #409] Added scroll bars

### LINKED ISSUE #409: Add scrollbars for dropdowns
It's overflowing and not shows some links in the dropdown lists.
image

image

### Add scrollbars like in https://github.com/fossasia/2019.fossasia.org/pull/35

# Am I working?
Yes of course!

### PR DESCRIPTION
**Pull request by supunTE**
Added scroll bars to dropdowns 
Fixed #409 
image
Preview - https://stedirisuriya.github.io/fossasia.org/

### CODE REVIEW

**Review by sarnava1 — APPROVED** [review]
LGTM

**Review by rpotter12 — APPROVED** [review]

File: about/index.html [hunk 1]
```diff
@@ -115,7 +115,7 @@
 							<li><a target="_self" href="../#subscribe">Subscribe</a></li>
 							<li><a target="_self" href="https://blog.fossasia.org">Blog</a></li>
 							<li class="has-dropdown"><a href="https://events.fossasia.org">Events</a>
-								<ul class="nav-dropdown" style="min-height: 575px">
+								<ul class="nav-dropdown" style="min-height: 575250pxpx">
```
Comment (championpaddler) [inline about/index.html]: @STEdirisuriya  Why are there two different values??

**Review by championpaddler — CHANGES_REQUESTED** [review]

File: about/index.html (hunk 1 above)
Comment (supunTE) [reply about/index.html]: @sk9331657 I am really sorry. It's a small typo. I fixed it now.

**Review by jogendra — APPROVED** [review]
Looks good to me.

**Review by samagragupta — APPROVED** [review]

**Review by nikit19 — APPROVED** [review]

**Comment by jogendra** [comment]
@mariobehling Please merge.

**Comment by mariobehling** [comment]
Not suitable. Rather implement sub menus.

**Pull request closed without merging**